lib/streamlit/errors.py

```diff
@@ -100,6 +100,31 @@ class DuplicateWidgetID(StreamlitAPIException):
     pass
 
 
+class StreamlitDuplicateElementID(DuplicateWidgetID):
```

Comment:
nit: `StreamlitDuplicateElementID` -> `StreamlitDuplicateElementId` (I would suggest to follow the same lowercasing guide we have started to adopt in our frontend based on [Google's style guide](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined))

Reply:
Changed 👍